configs/opencv-ovms/scripts/run_grpc_go.sh

```diff
@@ -20,7 +20,19 @@ fi
 
 echo $rmDocker
 
-containerDisplayPort=8080
-displayPortNum=$(( $cid_count + $containerDisplayPort ))
-echo "displayPortNum=$displayPortNum"
-docker run --network host --privileged $rmDocker -e displayPortNum="$displayPortNum" -e inputsrc="$inputsrc" -e cid_count="$cid_count" -e GRPC_PORT="$GRPC_PORT" -e DEBUG="$DEBUG" -v $RUN_PATH/results:/tmp/results --name dev"$cid_count" grpc:dev
\ No newline at end of file
+DOCKER_ENTRY="${PipelineStreamDensityRun:=./entrypoint.sh}"
+
+echo "DOCKER_ENTRY: $DOCKER_ENTRY"
+
+if [ "$STREAM_DENSITY_MODE" == 1 ]; then
+	stream_density_envs="-e STREAM_DENSITY_FPS=$STREAM_DENSITY_FPS -e STREAM_DENSITY_INCREMENTS=$STREAM_DENSITY_INCREMENTS -e COMPLETE_INIT_DURATION=$COMPLETE_INIT_DURATION"
+fi
+
+docker run --network host --privileged $rmDocker \
+	-e inputsrc="$inputsrc" -e cid_count="$cid_count" -e GRPC_PORT="$GRPC_PORT" -e DEBUG="$DEBUG" \
+	$stream_density_envs \
+	-v $RUN_PATH/results:/tmp/results \
+	-v $RUN_PATH/configs/dlstreamer/framework-pipelines/stream_density.sh:/home/pipeline-server/stream_density_framework-pipelines.sh \
```

Comment:
It is interesting that this is volume mount to configs/**dlstreamer**/framework-pipelines/stream_density.sh, so opencv-ovms doesn't have separated stream_density.sh? this is still universal for both dlstreamer and opencv-ovms?

Reply:
yes, we are using the same stream density script for both